apps/workflow-server/gcc-workflow-server-facade/gcc-restclient-facade/src/main/java/com/coremedia/labs/translation/gcc/facade/GCExchangeFacade.java

```diff
@@ -52,15 +52,18 @@
    * Submit submission for the given contents uploaded before.
    *
    * @param subject      workflow subject
+   * @param comment      instructions for translators
    * @param dueDate      due date for the submission; implies the priority when translation jobs should be done
+   * @param workflow     translation workflow to be used, if not the default
+   * @param submitter    name of the submitter
```

Comment:
The new parameters can be null, right? Should be mentioned in the description that specifying values is optional